src/command_manager.cc

```diff
@@ -51,20 +53,32 @@ command_manager& command_manager::instance() {
 void command_manager::enqueue(std::packaged_task<int(void)>&& f) {
   std::lock_guard<std::mutex> lock(_queue_m);
   _queue.emplace_back(std::move(f));
+  if (!_has_data) {
+    _has_data = true;
+    _queue_cv.notify_all();
+  }
 }
 
-void command_manager::execute() {
+void command_manager::execute(long time) {
```

Comment:
In the header of this fonction, add a doxygen message to tell what is this function and what is the needed argument. Give the unit of time also.